vagrant/roles/subman-devel/tasks/main.yml

```diff
@@ -176,6 +176,43 @@
     line: "PYTHONPATH={{ subman_checkout_dir }}/src:{{ subman_checkout_dir }}/python-rhsm/src"
   become: yes
 
+- name: compile c artifacts
+  make:
+    chdir: "{{ subman_checkout_dir }}"
+  when: subman_setup_hacking_environment and ansible_distribution == 'Debian'
+
+- name: install c artifacts
+  make:
+    chdir: "{{ subman_checkout_dir }}"
+    target: install
+  become: yes
+  when: subman_setup_hacking_environment and ansible_distribution == 'Debian'
+
+- name: configure python sys.path
```

Comment:
I think, this 'workaround' is, what enables me to run rhsmd by triggering dbus events. I think, DBus is not going to have the `$PYTHONPATH`. But maybe it is better to run them by hand, because restarting them after changing the source would be easier.
I am happy to leave the choice to you. Maybe you could add a small section to the README that the service must be started before subscription-manager can run properly.

Reply:
I thought, I needed to restart the dbus-service, but i just forgot to run `vagrant rsync-auto`.
This leaves me quite happy with the solution at hand.